android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -167,6 +188,20 @@
     private static final List<String> yandexRegions =
             Arrays.asList("AM", "AZ", "BY", "KG", "KZ", "MD", "RU", "TJ", "TM", "UZ");
 
+    public final class DialogOption {
```

Comment:
`class DialogOption` is not used

Reply:
i think that may be leftover from me and it appears to be added by Deep's commit because of a rebase, maybe

Reply:
`class DialogOption` is still here

Reply:
@AlexeyBarabash i was not sure if i should remove it from my PR as it was not introduced here. @alexsafe should i remove it from here ?

Reply:
yeap. i think it's fine to remove that part